src/iop/basecurve.c

```diff
@@ -329,7 +329,7 @@ static void set_presets(dt_iop_module_so_t *self, const basecurve_preset_t *pres
     dt_gui_presets_update_ldr(_(presets[k].name), self->op, self->version(), FOR_RAW);
     // make it auto-apply for matching images:
     dt_gui_presets_update_autoapply(_(presets[k].name), self->op, self->version(),
-                                    force_autoapply ? 1 : presets[k].autoapply);
+                                    (force_autoapply) 1 : (presets[k].autoapply && force_autoapply));
```

Comment:
this can't be correct, your second force_autoapply is always 0 so this is equivalent to:

   force_autoapply ? 1 : 0

Reply:
You are right and I need to wake-up. The last commit in master fixes it. Well done ! :+1: